src/components/internal/DatePicker/DatePicker.tsx

```diff
@@ -1,26 +1,34 @@
 import { DayPicker, Matcher } from "react-day-picker";
 import { Day } from "src/components/internal/DatePicker/Day";
-import { Header } from "src/components/internal/DatePicker/Header";
+import { Header, PreciseDateHeader, YearSkipHeader } from "src/components/internal/DatePicker/Header";
 import { WeekHeader } from "src/components/internal/DatePicker/WeekHeader";
 import { Css } from "src/Css";
 import { useTestIds } from "src/utils";
 import "./DatePicker.css";
 
+export type YearPicker = "default" | "skip" | "precise";
```

Comment:
**suggestion:** I would push for the `precise` version to replace `skip`. We'll need to chat with the designer to see if they are good with that too.